docs/installation/getting-started/quick-start.md

```diff
@@ -0,0 +1,313 @@
+
+### Toolbar composition
+
+Next step allows you to compose the toolbar. A simple drag-and-drop workspace allows for adding buttons (representing the plugins chosen in the previous step) to the toolbar. You may also change the order of the buttons and dropdowns and group them accordingly. Note that online builder allows you to create a multiline toolbar layout, too &mdash; just drag any button below the already placed ones to create a new toolbar line.
+
+{@img assets/img/online-builder-04-toolbar-configurator.png 753 The toolbar drag-and-drop configurator.}
+
+Some of the buttons are pre-placed on the layout and thus grayed out in the workspace with available toolbar items. If you want to remove any buttons from your toolbar setup, drag them back to the upper workspace.
+
+Once you finish designing the toolbar, press the **Next step** button on the top right.
+
+### Choosing the default language
+
+Scroll the list of available languages and check the one you want to be the main language of your editor build.
+
+{@img assets/img/online-builder-05-language-selection.png 367 Language selector list.}
+
+<info-box hint>
+	All other languages will still be available in the `translations` folder.
+</info-box>
+
+### Download the customized build
+
+This is as simple as it gets: just press the **Start** button to download your customized package.
+
+Refer to the *What's next?* note at the end of this page for further steps in your CKEditor 5 journey!
+
+## Building the editor from source
+
+This scenario allows you to fully control the building process of CKEditor 5. This means that you will not actually use the builds introduced in the previous path, but instead build CKEditor from source directly into your project. This integration method gives you full control over which features will be included and how webpack will be configured.
+
+This is an advanced path that assumes that you are familiar with npm and your project uses npm already. If not, see the [npm documentation](https://docs.npmjs.com/getting-started/what-is-npm) or call `npm init` in an empty directory and check the result.
+
+### Setting up the environment
+
+Before moving to the integration, you need to prepare three files that will be filled with code presented in this guide. Create the `webpack.config.js`, `app.js` and `index.html` files.
+
+Then, install packages needed to build CKEditor 5:
+
+```bash
+npm install --save \
+	postcss-loader@3 \
+	raw-loader@3 \
+	style-loader@1 \
+	webpack@4 \
+	webpack-cli@3
+```
+
+The minimal webpack configuration needed to enable building CKEditor 5 is:
+
+```js
+// webpack.config.js
+
+'use strict';
+
+const path = require( 'path' );
+const { styles } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+module.exports = {
+	// https://webpack.js.org/configuration/entry-context/
+	entry: './app.js',
+
+	// https://webpack.js.org/configuration/output/
+	output: {
+		path: path.resolve( __dirname, 'dist' ),
+		filename: 'bundle.js'
+	},
+
+	module: {
```

Comment:
Webpack 5 requires changes in the `module` object. See:
https://github.com/ckeditor/ckeditor5/blob/0a9e5a9c34a83324fec2242f2d5c86669c08ed02/packages/ckeditor5-build-classic/webpack.config.js#L60-L93